### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by jensschuett**
Should solve Issue #16

### CODE REVIEW

File: minify-html.php [hunk 1]
```diff
@@ -51,7 +51,7 @@ public function onPluginsInitialized()
   public function onOutputGenerated()
   {
     // Check if the page type is HTML
-    if ($this->grav['uri']->extension() === 'html') {
+    if ($this->grav['uri']->extension('html') === 'html') {
```
Comment (rogeriocosta-mb) [inline minify-html.php]: This actually sets the extension to HTML, it is not the best solution. I suggest test the page content type by change  the method to ```$this->grav['page']->templateFormat() === 'html'```

Comment (rogeriocosta-mb) [reply minify-html.php]: The final code would be ```if ($this->grav['page']->templateFormat() === 'html') {```

Comment (jensschuett) [reply minify-html.php]: Thank you, I wonder why the return type is null: https://github.com/getgrav/grav/blob/develop/system/src/Grav/Common/Page/Page.php#L1371

**Comment by jensschuett** [comment]
see #18

File: minify-html.php (hunk 1 above)
Comment (rogeriocosta-mb) [reply minify-html.php]: The null on this specific line represents that it is the default returned value :-)

Comment (jimblue) [reply minify-html.php]: thanks everyone!

**Pull request closed without merging**